### DISCUSSION TITLE
Using multiple languages, but no translations

### DISCUSSION THREAD

**Discussion by domma** [getpelican/pelican#3287] [Q&A]
Hi,

I'm quite happy with Pelican and how easy it is to customize it according to my needs. There is one thing I could not yet solve:

I publish blog posts in multiple (i.e. two) languages: IT related post are in English, other topics mostly in German, my native language. If I choose "en" as default language and set "Lang: de" in an article, the translation feature of Pelican kicks in, which results in a URL with "-de" appended. I don't want that. I just want to set the proper `html lang="de"` in my output.

Is there a way to simply disable the translation feature completely?

Achim

**Comment by avaris** [comment]
See various [`*_LANG_*` setings](https://docs.getpelican.com/en/4.9.1/settings.html#url-settings), particularly `ARTICLE_LANG_URL` and `ARTICLE_LANG_SAVE_AS`. Those define the URL/SAVE_AS for content other than default language. If you do not want the `lang` suffix, set them same as the non-LANG counterparts.
